addons/account/tests/test_account_account.py

```diff
@@ -342,3 +344,40 @@ def test_account_name_onchange(self):
         # saving a form without a code should not be possible
         with self.assertRaises(AssertionError):
             account_form.save()
+
+    @freeze_time('2023-09-30')
+    def test_generate_account_suggestions(self):
+        """
+            Test the generation of account suggestions for a partner.
+
+            - Creates: partner and a account move of that partner.
+            - Checks if the most frequent account for the partner matches created account (with recent move).
+            - Sets the account as deprecated and checks that it's no longer appears in the suggestions.
+
+            * since tested function takes into account last 2 years, we use freeze_time to (lock cpu time and to) make
+            sure the test will work independently of the date
```

Comment:
```suggestion
            * since tested function takes into account last 2 years, we use freeze_time
```
Can just explain why you use it, you don't really need to explain what it's doing